src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php

```diff
@@ -153,7 +153,7 @@ protected function file($file, string $fileName = null, string $disposition = Re
     protected function addFlash(string $type, string $message)
     {
         if (!$this->container->has('session')) {
-            throw new \LogicException('You can not use the addFlash method if sessions are disabled.');
+            throw new \LogicException('You can not use the addFlash method if sessions are disabled. Enable them in config/packages/framework.yaml.');
```

Comment:
This is not consistent with other message (no quotes but dot at the end).

Reply:
fixed